cxplat/cxplat_test/cxplat_fault_injection.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#if !defined(CMAKE_NUGET)
+#include <catch2/catch_all.hpp>
+#else
+#include <catch2/catch.hpp>
+#endif
+
+#include "cxplat_fault_injection.h"
+
+struct _on_exit
+{
+    std::function<void()> _func;
+    _on_exit(std::function<void()> func) : _func(func) {}
+    ~_on_exit() { _func(); }
+};
+
+enum class _fault_injection_test_state
+{
+    ExpectedInjectFault,               /// < Fault should be injected.
+    ExpectedNotInjectFault,            /// < Fault should not be injected.
+    ExpectetNotInjectFaultAfterReload, /// < Fault should not be injected after reloading.
```

Comment:
```suggestion
    ExpectedNotInjectFaultAfterReload, /// < Fault should not be injected after reloading.
```
Although I might find "Dont" instead of "Not" more readable.